### ISSUE TITLE
Wanted>Search All in "Missing" or "CutOff Unmet" process fails and breaks search function (Using NZBGet)

### ISSUE THREAD

**Issue by ChiefZigZag** [Radarr/Radarr#2022]
When running Search all the process ends and unable to search NZBs,and search function stops working completely. 
[v0.2.0.846] System.Net.WebException: Error: SecureChannelFailure (The authentication or decryption has failed.) ---> System.IO.IOException: The authentication or decryption has failed.
[radarr.txt](https://github.com/Radarr/Radarr/files/1264658/radarr.txt)

**Comment by galli-leo** [comment]
What version of mono are you using?

**Comment by ChiefZigZag** [comment]
Let me know if you need any help with this problem, I am willing to commit time and resources into troubleshooting if need be. Keep up the good work.

**Comment by ChiefZigZag** [comment]
Update: I tested it out again today, I did a "Search All" under "Cutoff Unmet" it successfully completed. However, Doing a "Search All" on the "Missing" section immediately failed. The health section is displaying the following error "All search-capable indexers are temporarily unavailable due to recent indexer errors"..doesn't make any sense since I am able to test the indexer and it works fine, works for the "cutoff unmet". I am using NZBHydra as well, works well for everything else "Sonarr, Headphones".

**Comment by galli-leo** [comment]
Can you try upgrading to mono 5.0 / try using your indexers over http? Seems like the https stuff is failing.

**Comment by ChiefZigZag** [comment]
The weird thing is I have been using the indexer over http, It has never been configured for https. I also am using FreeBSD 11.1 and I checked to see if I could easily upgrade mono to 5.0 but it seems there is no port for it available at the moment. I'll see what happens when I configure the indexer for https and see if it makes any sort of difference.

**Comment by ChiefZigZag** [comment]
Found a solution to a similar report on Sonarr. "In the configuration I have 2 nzb access type configuration.
1) Redirect to the indexer (the default one)
2) Proxy NZB from the indexer
I used the Proxy NZB option and it's works! I also added the environment variable in my systemd because I'm now with Mono 5." https://forums.sonarr.tv/t/securechannelfailure-the-authentication-or-decryption-has-failed-nzbhydra/15489/14
